src/instructlab/model/evaluate.py

```diff
@@ -31,7 +33,7 @@ class Benchmark(str, enum.Enum):
     MMLU_BRANCH = "mmlu_branch"
     MT_BENCH = "mt_bench"
     MT_BENCH_BRANCH = "mt_bench_branch"
-
+    LLMAAJ = "llmaaj"
```

Comment:
LLMAAJ is a very broad term. We are specifically doing open response QA eval. So I would either call this QA grading or RAG eval.